launchflow/workflows/k8s_service.py

```diff
@@ -0,0 +1,79 @@
+from typing import Dict, Optional
+from kubernetes import client
```

Comment:
maybe we should install kubernetes separately, similar to launchflow[gcp] and launchflow[aws] ?

Reply:
looks like its only 1MB so maybe its fine to always have it included

Reply:
yea it's got pretty minimal dependencies too, and I don't think we'd ever have more than one in the group so probably not worth it